### PR TITLE
Configure kotlindoc generation per SDK.

### PR DESCRIPTION
**Pull request by vkryachko**
This makes it easier to generate docs for a subset of SDKs, additionally
it produces a set of separate _toc.yaml files as opposed to a global one.

### CODE REVIEW

File: buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/Dokka.java [hunk 1]
```diff
@@ -0,0 +1,179 @@
+// Copyright 2019 Google LLC
```
Comment (rlazo) [inline buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/Dokka.java]: 2020 maybe :-)

Comment (vkryachko) [reply buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/Dokka.java]: Done

**Pull request merged**